charts/newrelic-infrastructure-beta/values.yaml

```diff
@@ -0,0 +1,389 @@
+# Default values for newrelic-infrastructure.
+nameOverride: ""
+fullnameOverride: ""
+
+images:
+  pullSecrets: []
+  forwarder:
+    pullPolicy: IfNotPresent
+    tag: 1.20.5
+    repository: newrelic/k8s-events-forwarder
+    registry: docker.io
+  agent:
+    pullPolicy: IfNotPresent
+    tag: 2.7.6
+    repository: newrelic/infrastructure-bundle
+    registry: docker.io
+  scraper:
```

Comment:
Since we've gotten rid of the `scraper` name elsewhere, should we change it here as well? Something like `nri-kubernetes`?